packages/contracts/scripts/deploy/dollar/solidityScripting/05_StakingShare.s.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import "./04_UbiquityCredit.s.sol";
+
+contract StakingShareScript is CreditScript {
+    function run() public virtual override {
+        super.run();
+        vm.startBroadcast(deployerPrivateKey);
+
+        // grant diamond token minter rights
+        IAccessCtrl.grantRole(STAKING_SHARE_MINTER_ROLE, address(diamond));
+        // add staking shares
+        string
+            memory uri = "https://bafybeifibz4fhk4yag5reupmgh5cdbm2oladke4zfd7ldyw7avgipocpmy.ipfs.infura-ipfs.io/";
```

Comment:
I just remembered this doesn't work. We should consider hosting off-chain and on our own website!

Reply:
Sure what URI should we use ?

Reply:
I'll fix it